### PR TITLE
Feature: Display trainees in cohort

### PR DESCRIPTION
**Pull request by alaintresor**
# PR Description

It displays every trainee include in a cohort

# Description of tasks that were expected to be completed

- Create a new trainee modal that displays all trainees in a cohort 

# How has this been tested?

- Login as admin
- Go to the Cohort menu
- Click on the people icon to get all trainees that include in that cohort

# Number of Commits

1 commit

# Screenshots (If appropriate)
Screenshot (3)
Screenshot (2)

# Please check this Checklist before you submit your PR:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My code generate no warnings
- [ ] My test coverage meet the set test coverage threshold
- [ ] There are no vulnerabilities
- [ ] There are no conflicts with the base branch

### CODE REVIEW

**Comment by Procech20** [comment]
I deployed a review app for you to try out this pull request 👇🏽
Heroku App: https://devpulse-review-app-143.herokuapp.com/


<hidden purpose="for-rewritable-pr-comment-action-use" value="Deploy"></hidden>

**Review by Placide123 — APPROVED** [review]

**Review by alicepretty — APPROVED** [review]

**Review by Dev-nkundineza — APPROVED** [review]

**Review by Procech20 — APPROVED** [review]

**Review by Procech20 — DISMISSED** [review]
Am getting an error that says "Not logged in an organization yet I did"!

**Comment by alaintresor** [comment]
>



> Am getting an error that says "Not logged in an organization yet I did"!

It's now fixed, please try it again

File: src/containers/admin-dashBoard/CohortTraineeModal.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        orgToken: organizationToken,
+        cohort: currentCohort ? currentCohort.name : '',
+      },
+    },
+  );
+
+  if (traineeData && traineeData.length > 0) {
+    traineeData?.map((data: any, index: number): any => {
+      datum[index] = {};
+      datum[index].name = data.profile ? data.profile.name : 'undefined';
```
Comment (ceelogre) [inline src/containers/admin-dashBoard/CohortTraineeModal.tsx]: Why did we re-introduce `undefined` for profile names?

**Review by ceelogre — COMMENTED** [review]
You can respond to my comment then we proceed with merging.


Also, the modal does not fit the whole width of the screen. I'm not sure if it was on purpose.
image

**Pull request closed without merging**